src/Conticket/Model/Aggregates/Event/Event.php

```diff
@@ -53,4 +60,23 @@ public function whenEventWasCreated(EventWasCreated $eventWasCreated)
         $this->name = $eventWasCreated->name();
         $this->description = $eventWasCreated->description();
     }
+
+    public function addTicket($name, $description)
+    {
+        $ticketId = new TicketId();
+        $this->recordThat(TicketWasAdded::fromTicketAndNameAndDescription(
+            $ticketId->toString(),
+            $name,
+            $description
+        ));
+    }
+
+    public function whenTicketWasAdded(TicketWasAdded $ticketWasAdded)
+    {
+        $this->tickets[] = Ticket::fromIdAndNameAndDescription(
```

Comment:
I didn't get it, I should construct Ticket under the TicketWasAdded event? This method already is where event is received